hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/replication/VerifyReplication.java

```diff
@@ -788,6 +852,23 @@ private static void printUsage(final String errorMsg) {
         + "2181:/cluster-b \\\n" + "     TestTable");
   }
 
+  private static ExecutorService buildReCompareExecutor(int maxThreads, Mapper.Context context) {
```

Comment:
I'm not entirely certain when `cleanup` gets called. Does it make sense to only build the reCompareExecutor if it is null? Or do we want to build a new executor on every call to `map`?